app/javascript/chat/actions/requestActions.js

```diff
@@ -0,0 +1,71 @@
+export function rejectJoiningRequest(
+  channelId,
+  membershipId,
+  successCb,
+  failureCb,
+) {
+  fetch(`/chat_channel_memberships/remove_membership`, {
```

Comment:
you could use the new `http/request` util, like this:

```js
import { request } from '@utilities/http';
const response = await request('/path');
```

it'll greatly simplify this code.

see https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/utilities/http/request.js

Reply:
Will do it.